### PR TITLE
Add Fanatical.com to Gaming

### PR DESCRIPTION
**Pull request by webdevian**

### CODE REVIEW

**Review by Jawshy — DISMISSED** [review]
Thanks for the submission, @webdevian.

File: _data/gaming.yml [hunk 1]
```diff
@@ -97,6 +97,13 @@ websites:
     tfa:
       - software
     doc: https://support.faceit.com/hc/en-us/articles/115000254670
+    
+  - name: Fanatical
+    url: https://www.fanatical.com
+    img: fanatical.png
+    tfa:
+      - software
```
Comment (kmpoppe) [inline _data/gaming.yml:105]:
```suggestion
      - totp
```
This would be `totp` not just simply `software` from what I see in the doc.

**Review by kmpoppe — CHANGES_REQUESTED** [review]
See inline comments, please :)

**Comment by kmpoppe** [comment]
@webdevian As you have not allowed maintainer edits to your branch I can't commit my suggestions myself. May I ask you to change the 2fa-method according to my suggestion?
If we don't hear back from you within 2 weeks, I will close this PR due to inactivity.
Thank you for your contribution and your understanding!
// Kai

**Comment by webdevian** [comment]
Sorry I missed these comments, I have created a new PR

**Pull request closed without merging**